src/main/java/com/theironyard/war/Deck.java

```diff
@@ -0,0 +1,45 @@
+package com.theironyard.war;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by EddyJ on 6/23/16.
+ */
+public class Deck {
+    private List<Card> cards = new ArrayList<Card>();
+
+    public Deck(){
```

Comment:
There is no need for a blank constructor.  If it doesn't do anything then just remove it and let the default `Object` constructor take over.